examples/vite-react-with-typescript/.eslintrc.cjs

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
Added a vite project to allow us to test standard setups for client-side only React applications. vite has replaced "create react app" as the de facto build tool for client side react applications.